intellst-front/public/index.html

```diff
@@ -9,6 +9,7 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
   </head>
+  <body>
```

Comment:
why do you want another body? the one you've got is already great! ;)

Reply:
Fixed